_layouts/product.html

```diff
@@ -183,6 +183,18 @@
   <td {% if diff <= 0 %} class = "txt-linethrough" {% endif %} >      <!-- if the support finished add txt-linethrough class -->
     {% if releaseLink != "" %}
       <a href="{{releaseLink}}" title="Release Notes / Changelog">{{latestVersionNumber}}</a>
+{% if  diff > 1 %}
+      &ensp;
+      <a href="http://www.google.com/calendar/render?action=TEMPLATE
+&text=[endoflife.date] {{page.title}} {{r.releaseCycle}} EOL reached
+&dates={{r.eol | replace:'-',''}}T000000/{{r.eol | replace:'-',''}}T000000
```

Comment:
actually i prefered it as on purpouse because it will let users point a new day's start which is pointing its eol

Reply:
is it okey to mark this done ? @marcwrobel

Reply:
Personally I think you achieve the same purpose if you do not specify the time. And in this context I hope it will not cause timezone issues (I experienced a lot of issues with time / timezone management so I tend to be overly cautious ;)).

But if you still prefer mentioning the time in the event, yes you can mark this conversation as done.